features/steps/cli_steps.py

```diff
@@ -453,6 +478,49 @@ def is_created(name):
         assert is_created(path)
 
 
+@then('the expected tool directories and files should be created with "{tools}"')
+def check_created_project_structure_from_tools(context, tools):
+    """Behave step to check the subdirectories created by kedro new with tools."""
+
+    def is_created(name):
+        """Check if path exists."""
+        return (context.root_project_dir / name).exists()
+
+    # Base checks for any project
+    for path in ["README.md", "src", "pyproject.toml", "requirements.txt"]:
+        assert is_created(path), f"{path} does not exist"
+
+    tools_list = (
+        tools.split(",") if tools != "all" else ["1", "2", "3", "4", "5", "6", "7"]
+    )
+
+    if "1" in tools_list:  # lint tool
+        pass
+
+    if "2" in tools_list:  # test tool
+        assert is_created("tests"), "tests directory does not exist"
+
+    if "3" in tools_list:  # log tool
+        assert is_created("conf/logging.yml"), "logging configuration does not exist"
+
+    if "4" in tools_list:  # docs tool
+        assert is_created("docs"), "docs directory does not exist"
+
+    if "5" in tools_list:  # data tool
+        assert is_created("data"), "data directory does not exist"
+
+    if "6" in tools_list:  # PySpark tool
+        assert is_created("conf/base/spark.yml"), "spark.yml does not exist"
+
+    if "7" in tools_list:  # 'viz' tool
```

Comment:
```suggestion
    if "7" in tools_list:  # viz tool
```
Nit: For consistency :)